modules/scripts/startup-script/files/install_ansible.sh

```diff
@@ -172,7 +172,19 @@ main() {
 	# Create pip virtual environment for HPC Toolkit
 	${python_path} -m pip install virtualenv
 	${python_path} -m virtualenv /usr/local/ghpc-venv
-	python_path=/usr/local/ghpc-venv/bin/python3
+	venv_python_path=/usr/local/ghpc-venv/bin/python3
+
+	# when Ansible creates virtual environments, it defaults to virtualenv
+	# these steps install virtualenv in the Ansible virtual environment and
+	# ensures that it will be found
+	${venv_python_path} -m pip install virtualenv
+	if [ ! -f /etc/ansible/ansible.cfg ]; then
+		mkdir /etc/ansible
+		cat <<-EOF >/etc/ansible/ansible.cfg
+			[defaults]
+			interpreter_python=${venv_python_path}
+		EOF
```

Comment:
nit: Any chance you could throw the following in here while you are at it and improve the readability of our ansible logging.

```
stdout_callback=debug
stderr_callback=debug
```